packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -134,7 +132,7 @@ class Draggable<T> extends StatefulWidget {
 
 
   /// The data that will be dropped by this draggable.
-  final T data;
+  final T? data;
```

Comment:
shouldn't this be non-nullable? i guess this is the source of the `?`s I commented on above...

Reply:
Get ride of `?` means make data required in the constructor. Some tests create Draggable without data. Should we required data to create Draggable?

Reply:
cc @goderbauer any idea if any of our (non-test) customers (e.g. google3 or fuchsia) use Draggable without data?

Reply:
Quick scan in google3 seems to indicate that Draggable is never used without data. Not sure about fuchsia...